novawallet/Common/ViewModel/RemoteImageViewModel.swift

```diff
@@ -49,7 +49,10 @@ extension RemoteImageViewModel: ImageViewModelProtocol {
     }
 
     func cancel(on imageView: UIImageView) {
-        imageView.kf.cancelDownloadTask()
+        imageView.kf.cancelDownloadTask() // cancel any dowload task
+
+        let url: URL? = nil
+        imageView.kf.setImage(with: url) // cancel any cache retrieval task
```

Comment:
I think it can be shorter: imageView.kf.setImage(with: nil)

Reply:
I tried by there are several methods with the same names for the parameter by different types. So we need to specify type to avoid ambiguity.